client/modules/IDE/actions/collections.js

```diff
@@ -152,11 +146,7 @@ export function deleteCollection(collectionId) {
     return apiClient
       .delete(url)
       .then((response) => {
-        dispatch({
-          type: ActionTypes.DELETE_COLLECTION,
-          payload: response.data,
-          collectionId
-        });
+        dispatch(delCollection(response.data, collectionId));
```

Comment:
Action creators from `createSlice` always take exactly one argument which is the `payload` (unless using `prepare` but let's not get into that 😆).  It doesn't make sense to call it with with two arguments.  The second argument will be ignored.  It looks like in the reducer you are are destructuring the `collectionId` property off of the `response.data` object which is the `payload`.  If that's all you need then you can make the payload be the `collectionId` and call `dispatch(delCollection(collectionId));`